src/actions/playerActions.js

```diff
@@ -101,3 +101,26 @@ export function swapPlayers (player1Id, player2Id) {
       });
   }
 }
+
+export function savePlayer (playerId, playerName, slackName) {
+  return function (dispatch) {
+    dispatch({
+      type: ActionType.SET_IS_LOADING,
+      isLoading: true
+    })
+    axios.put(`/api/players/${playerId}`, {
+      playerName,
+      slackName
+    })
+      .then(() => {
+        dispatch({ type: ActionType.SAVE_PLAYER })
```

Comment:
After this line you can call `dispatch(getPlayers())` to get the updated values into the UI. (Otherwise I have to refresh the page in order to see the updated values.)